start.py

```diff
@@ -302,6 +303,31 @@ def activate_new_relic(m2ee, app_name):
 
 def set_up_m2ee_client(vcap_data):
     m2ee = M2EE(yamlfiles=['.local/m2ee.yaml'], load_default_files=False)
+    version = m2ee.config.get_runtime_version()
+
+    mendix_runtimes_path = '/usr/local/share/mendix-runtimes.git'
+    mendix_runtime_version_path = os.path.join(os.getcwd(), 'runtimes', str(version))
+    if os.path.isdir(mendix_runtimes_path) and not os.path.isdir(mendix_runtime_version_path):
+        buildpackutil.mkdir_p(mendix_runtime_version_path)
+        env = dict(os.environ)
+        env['GIT_WORK_TREE'] = mendix_runtime_version_path
+
+        # checkout the runtime version
+        process = subprocess.Popen(['git', 'checkout', str(version), '-f'], cwd=mendix_runtimes_path, env=env,
+                                   stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if process.returncode is not 0:
+            # do a 'git fetch --tags' to refresh the bare repo, then retry to checkout the runtime version
+            logging.info('mendix runtime version {mx_version} is missing in this rootfs'.format(mx_version=version))
+            process = subprocess.Popen(['git', 'fetch', '--tags', '&&', 'git', 'checkout', str(version), '-f'],
+                                       cwd=mendix_runtimes_path, env=env, stdout=subprocess.PIPE,
+                                       stderr=subprocess.PIPE)
+            if process.returncode is not 0:
```

Comment:
Hey,

Just a small note, on this line and any similar one, use **is** keyword **when and only when** you are comparing object identities. Use == when comparing values.